easybuild/easyblocks/g/gcc.py

```diff
@@ -1043,4 +1048,42 @@ def make_module_req_guess(self):
             'LD_LIBRARY_PATH': ['lib', 'lib64'],
             'MANPATH': ['man', 'share/man']
         })
+
+        # Install rpath wrappers, if built with RPATH enabled
+        if build_option('rpath'):
+            def create_rpath_wrappers(targetdir):
+                set_up_configuration(silent=True)
+                tc = Toolchain(name="system", version="0.1")
+
+                # Expect rpath_filter_dirs and rpath_include_dirs as environment variables of
+                # ,-separated lists by default. We might want to allow different separators.
+                # E.g. ":" could be used to get a similar syntax to PATH and LD_LIBRARY_PATH
+                separator = ','
+                rpath_filter_dirs = None
+                if "RPATH_FILTER_DIRS" in os.environ:
+                    rpath_filter_dirs = os.environ["RPATH_FILTER_DIRS"].split(separator)
+                rpath_include_dirs = None
+                if "RPATH_INCLUDE_DIRS" in os.environ:
+                    rpath_include_dirs = os.environ["RPATH_INCLUDE_DIRS"].split(separator)
+
+                tc.prepare_rpath_wrappers(
+                        rpath_filter_dirs=rpath_filter_dirs,
+                        rpath_include_dirs=rpath_include_dirs
+                        )
+
+                # Find location of rpath wrappers. Tempfile in prepare_rpath_wrappers seems to be
+                # in a parent directory of our current gettempdir()-location
+                wrapperpath = glob.glob(os.path.join(tempfile.gettempdir(), "..", "tmp*", RPATH_WRAPPERS_SUBDIR))[0]
+                move_file(wrapperpath, targetdir)
+
+            rpath_wrapperdir = os.path.join('bin', RPATH_WRAPPERS_SUBDIR)
+            absolute_wrapperdir = os.path.join(self.installdir, rpath_wrapperdir)
+            if not os.path.exists(absolute_wrapperdir):
+                # make_module_req_guess seems to be called multiple times
+                # create and move rpath wrappers only once!
+                create_rpath_wrappers(absolute_wrapperdir)
+            for wrapper in os.listdir(absolute_wrapperdir):
```

Comment:
I think we should aim to have all wrapper scripts directly in `bin/rpath_wrappers`, rather than in a subdirectory thereof?

That's what EasyBuild does dynamically, out of an abundance of caution, but it doesn't make much sense when the wrappers are included with `GCCcore` itself I think.

Reply:
This is another thing that should be addressed by the future easybuild-framework PR.
`prepare_rpath_wrappers` currently creates a single subdir per wrapper script, but it would be better to ship them in a single directory when installing the module.